db/migrate/20250603031148_add_owner_to_books.rb

```diff
@@ -0,0 +1,5 @@
+class AddOwnerToBooks < ActiveRecord::Migration[7.1]
+  def change
+    add_reference :books, :owner, foreign_key: { to_table: :users }
```

Comment:
This is normal rails way to tie 2 tables together, but we don't want admin users to own books, we just want books to have a list of editors. An array of strings would be good, or maybe a jsonb field to have e.g. {"regan" => "regan.ryan.nz@gmail.com"} yea that's better because then we can list the names of the owners, since some people have weird emails